src/service/url-replacements-impl.js

```diff
@@ -753,6 +193,7 @@ export function installUrlReplacementsServiceForDoc(ampdoc) {
  * @param {*} varSource
  */
 export function installUrlReplacementsForEmbed(ampdoc, embedWin, varSource) {
+  installVarSubstitutionForEmbed(ampdoc, embedWin, varSource);
   installServiceInEmbedScope(embedWin, 'url-replace',
-      new UrlReplacements(ampdoc, varSource));
+      new UrlReplacements(ampdoc));
```

Comment:
I believe it should be enough because service for doc is installed elsewhere. otherwise, will it be okay to call it here again? it should be a noop if the service is already installed.